### PR TITLE
What: Update Sandbox readme to explain process and community questions/comment expectations

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:
 * Current README does not provide sufficient information on expectations (projects or community members)

This change address the need by:
 * adding more info about what occurs between application and review meeting
 * provide information to projects on TAG, TOC, and community comments/questions
 * advise community members on appropriate comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ Applying is as easy as 1-2-3!
...
+
+### Community Comments on Applications
+
+The TOC welcomes and appreciates community members support and exploration in surfacing items for technical consideration as part of sandbox applications, it allows the TOC to have a more comprehensive perspective on each projects' potential and supports our TAGs in understanding more about these projects. 
+
+In keeping with the [Technical Leadership Principles](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#technical-leadership-principles), community members providing comments and querstions should [be welcoming and curious in their comments](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#be-welcoming), [provide feedback with courtesy](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#provide-feedback-and-appreciation-appropriately), and [foster respectful resolution](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#foster-respectful-resolution).  Embodying the technical leadership principles as you comment on these applications directly contributes to sustaining the open, community-oriented nature of CNCF.
```
Comment (angellk) [inline README.md:27]: @TheFoxAtWork small typo: 'questions'

**Review by angellk — APPROVED** [review]
One small nit, not a blocker. Thanks for adding this additional information @TheFoxAtWork

File: README.md (hunk 1 above)
Comment (TheFoxAtWork) [reply README.md]: fixed!  ( i really need to remember to spell check)

**Comment by dims** [comment]
LGTM

**Pull request merged**